src/Skia/Avalonia.Skia/DrawingContextImpl.cs

```diff
@@ -1263,15 +1267,7 @@ public void AddDisposable(IDisposable disposable)
             /// <inheritdoc />
             public void Dispose()
             {
-                if (_disposePaint)
```

Comment:
I think we should keep `_disposePaint` and return SKPaint to pool instead of disposing it. The flag was there to mark short-lived paint objects.

Reply:
It would also help with keeping `using` pattern for temporary SKPaint instances like one in PopOpacityMask

Reply:
> I think we should keep `_disposePaint` and return SKPaint to pool instead of disposing it. The flag was there to mark short-lived paint objects.

If I read the code link below correctly, this line below is the only reason for `PaintWrapper._disposePaint` to begin with. I see no other paths that use it.  If this is something for the future, I understand and will add it back. See my explanation in the next comment below.

https://github.com/AvaloniaUI/Avalonia/blob/24bc7a6887c90b57fce61b4662890ddd7abe764d/src/Skia/Avalonia.Skia/DrawingContextImpl.cs#LL627

Removing the _disposePaint` removes one extra byte off the struct and makes the struct a nice aligned 32 byes instead of 40 bytes.

The big intent of this PR is to reduce memory pressure and making the PaintWrapper 32 bit aligned was just a happy accident.  What are your thoughts?

> It would also help with keeping `using` pattern for temporary `SKPaint` instances like one in `PopOpacityMask`

I believe there is not a need of any temporary `SKPaint` instances.  See my comments below.